server/hedley/modules/custom/hedley_admin/scripts/generate-demographics-hc-report.php

```diff
@@ -11,13 +11,53 @@
 require_once __DIR__ . '/report_common.inc';
 
 $limit_date = drush_get_option('limit_date', FALSE);
-$region = drush_get_option('region', FALSE);
-
+$region_name = drush_get_option('region', FALSE);
 if (!$limit_date) {
   drush_print('Please specify --limit_date option');
   exit;
 }
 
+/**
+ * Gets the node ID of the health center.
+ *
+ * @param string $health_center_name
+ *   The name of a healthcenter.
+ *
+ * @return string
+ *   The node ID of the health center.
+ */
+function get_health_center_id($health_center_name = NULL) {
+  if ($health_center_name) {
+    $health_center_name = str_replace(['_', '-', '.'], ' ', $health_center_name);
+    $results = db_query("SELECT nid FROM node WHERE title='$health_center_name' AND type='health_center'");
+    if (!$results->fetchField()) {
+      $results = db_query("SELECT nid FROM node WHERE title LIKE '$health_center_name%' AND type='health_center'");
```

Comment:
This is more tricky:
https://api.drupal.org/api/drupal/includes!database!database.inc/function/db_like/7.x